modules/installation-nutanix-failure-domains-req.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/nutanix-failure-domains.adoc
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-nutanix-failure-domains-req_{context}"]
+= Failure domain requirements
+
+When planning to use failure domains, consider the following requirements:
```

Comment:
```suggestion
When planning to use failure domains in your Nutanix Prism Element (cluster), consider the following requirements:
```

Reply:
Or: 
```suggestion
When planning to use Nutanix failure domains, consider the following requirements:
```

Reply:
Failure domains are not something specific to Nutanix. This sentence is technical accurate as is.